README.md

```diff
@@ -18,210 +18,294 @@ Darkflame Universe is licensed under AGPLv3, please read [LICENSE](LICENSE). Som
 
-#### Setup locale
-* In the `build` directory create a `locale` directory if it does not already exist
-* Copy over or create symlinks from `locale.xml` in your client `locale` directory to the `build/locale` directory
 
-#### Client database
-* Move the file `res/cdclient.fdb` from the unpacked client to the `build/res` folder on the server.
-* The server will automatically copy and convert the file from fdb to sqlite should `CDServer.sqlite` not already exist.
-* You can also convert the database manually using `fdb_to_sqlite.py` using lcdr's utilities.  Just make sure to rename the file to `CDServer.sqlite` instead of `cdclient.sqlite`.
-* Migrations to the database are automatically run on server start.  When migrations are needed to be ran, the server may take a bit longer to start.
+## Build the server
 
-### Database
-Darkflame Universe utilizes a MySQL/MariaDB database for account and character information.
+### Enabling testing
+While it is highly recommended to enable testing, if you would like to save compilation time, you'll need to comment out the [enable_testing variable in CMakeVariables.txt](https://github.com/DarkflameUniverse/DarkflameServer/blob/main/CMakeVariables.txt#L21).
+It is recommended that after building and if testing is enabled, to run `ctest` and make sure all the tests pass.
 
-Initial setup can vary drastically based on which operating system or distribution you are running; there are instructions out there for most setups, follow those and come back here when you have a database up and running.
+### Building on MacOS
+In the repository root folder run the following. Ensure -DOPENSSL_ROOT_DIR=/path/to/openssl points to your openssl install location.  This is generally located in or around `/usr/local/opt/openssl@3/` but may differ.  For example, if openssl is located at `/usr/local/opt/openssl@3/` then the full cmake command would be `cmake .. -DOPENSSL_ROOT_DIR=/usr/local/opt/openssl@3/`
 
-* All that you need to do is create a database to connect to.  As long as the server can connect to the database, the schema will always be kept up to date when you start the server.
+```bash
+# Create the build directory, preserving it if it already exists
+mkdir -p build
+cd build
 
-#### Configuration
+# Run CMake to generate build files
+cmake .. -DOPENSSL_ROOT_DIR=/path/to/openssl
 
-After the server has been built there should be four `ini` files in the build director: `sharedconfig.ini`, `authconfig.ini`, `chatconfig.ini`, `masterconfig.ini`, and `worldconfig.ini`. Go through them and fill in the database credentials and configure other settings if necessary.
+# Get cmake to build the project. If make files are being used then using make and appending `-j` and the amount of cores to utilize may be preferable, for example `make -j8`
+cmake --build . --config Release
+```
 
-#### Migrations
+### Building on **all** other platforms
+You can either run `build.sh` when in the root folder of the repository:
```

Comment:
Is this really accurate for Windows?

Reply:
running the build script works on Windows